natgeo/mortar
# AEM-6914+AEM-6818 Remove Broadsheet Border Animation and update border widths

https://jira.natgeo.com/browse/AEM-6818
https://jira.natgeo.com/browse/AEM-6914

This work removes the existing animation for the border of the broadsheet component as well as updates the width of the border at various breakpoints. The changing of the width of the border at the various breakpoints necessitated the refactoring of some layout rules.

To test:

- Pull AEM-6914+AEM-6818
- Run npm test to verify css still passes linting
- Run gulp serve
- Verify Broadsheet component layout at various breakpoints
